src/cluster.c

```diff
@@ -6803,7 +6803,7 @@ void clusterRedirectClient(client *c, clusterNode *n, int hashslot, int error_co
         /* The request spawns multiple keys in the same slot,
          * but the slot is not "stable" currently as there is
          * a migration or import in progress. */
-        addReplyError(c,"-TRYAGAIN Multiple keys request during rehashing of slot");
+        addReplyError(c,"-UNSTABLE TRYAGAIN Multiple keys request during rehashing of slot");
```

Comment:
This will break clients. The error should be -TRYAGAIN and it is also what's documented in the cluster specification:

> Operations on keys that don't exist or are - during the resharding - split between the source and destination nodes, will generate a `-TRYAGAIN` error. The client can try the operation after some time, or report back the error.

The C constant is called CLUSTER_REDIR_UNSTABLE, but that's just an internal name. The clients see only -TRYAGAIN.

Reply:
> This will break clients. The error should be -TRYAGAIN and it is also what's documented in the cluster specification:
> 
> > Operations on keys that don't exist or are - during the resharding - split between the source and destination nodes, will generate a `-TRYAGAIN` error. The client can try the operation after some time, or report back the error.
> 
> The C constant is called CLUSTER_REDIR_UNSTABLE, but that's just an internal name. The clients see only -TRYAGAIN.

@zuiderkwast Thanks for your comments, I roll back to previous error message, and at the same time, I decrease the numner of keys to 5000 to test, which was 40000.